ksqldb-parser/src/test/java/io/confluent/ksql/util/QueryMaskTest.java

```diff
@@ -0,0 +1,279 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.util;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+import org.junit.Test;
+
+public class QueryMaskTest {
+
+  @Test
+  public void shouldMaskConfigMap() {
+    // Given
+    final ImmutableMap<String, String> config = ImmutableMap.of(
+        "connector.class", "someclass",
+        "model", "somemode",
+        "key", "somekey"
+    );
+
+    // When
+    final Map<String, String> maskedConfig = QueryMask.getMaskedConnectConfig(config);
+
+    // Then
+    final ImmutableMap<String, String> expectedConfig = ImmutableMap.of(
+        "connector.class", "someclass",
+        "model", "'[string]'",
+        "key", "'[string]'"
+    );
+
+    assertThat(maskedConfig, is(expectedConfig));
+  }
+
+  @Test
+  public void shouldMaskIfNotExistSourceConnector() {
```

Comment:
This version doesn't support IF NOT EXISTS, does it? This test looks like it needs to be removed.